api/src/dto/ChangeDto.ts

```diff
@@ -0,0 +1,21 @@
+import { DocType, Uuid } from "../types";
+import { IsNotEmpty, IsObject } from "class-validator";
+import { GroupAclEntryDto } from "./GroupAclEntryDto";
+
+/**
+ * Database structured Post object
+ */
+export class ChangeDto {
+    @IsNotEmpty()
+    _id: Uuid;
+    @IsNotEmpty()
+    type: DocType.Change;
```

Comment:
Does this automatically check if the type is "change"?

Reply:
Adding a Equals decorator to ensure the correct value